00-intro.md

```diff
@@ -11,15 +11,15 @@ minutes: 5
 
 At a high level, computers do four things:
 
--   run programs
--   store data
--   communicate with each other
--   interact with us
+-   run programs,
+-   store data,
+-   communicate with each other,
+-   and interact with us.
```

Comment:
Many style guides (like this one https://proswrite.com/2013/06/20/confused-about-punctuation-in-bullet-lists/) suggest punctuating the list unless they consist of single words:

> If the stem sentence is an introductory phrase or clause, followed by a list, we start each list item with a lower case letter, and leave end punctuation off each item but the last. We give the last one a full stop. In essence, we see the list as a run-on sentence, with white space taking the place of punctuation between each item in the list.